packages/govern-console/src/containers/DAO/components/DaoSideCard/DaoSideCard.tsx

```diff
@@ -1,9 +1,119 @@
+import {
+  Box,
+  EthIdenticon,
+  IconActionsFilled,
+  IconSettingsFilled,
+  IconFinanceFilled,
+  useLayout,
+} from '@aragon/ui';
+import styled from 'styled-components';
+
+import ActionLink from '../ActionLink/ActionLink';
+
 type Props = {
+  address: string;
   baseUrl: string;
+  identifier: string;
+  openActions: string;
+};
+
+type MenuOptions = {
+  path: string;
+  label: string;
+  icon: React.ReactElement;
 };
 
-const DaoSideCard: React.FC<Props> = ({ baseUrl }) => {
-  return <div>DAO Side Card: {baseUrl}</div>;
+const links: MenuOptions[] = [
+  { path: '/actions', label: 'Actions', icon: <IconActionsFilled size="small" /> },
+  { path: '/finance', label: 'Finance', icon: <IconFinanceFilled size="small" /> },
+  { path: '/settings', label: 'Settings', icon: <IconSettingsFilled size="small" /> },
+];
+
+const Container = styled.div`
+  border: 1px solid transparent;
+  position: relative;
+`;
+
+const AvatarContainer = styled.div`
+  position: absolute;
```

Comment:
FYI: you can (and should 😬) have props in styled-components! 🙂

For example, you could change this as follows: 

```ts
const AvatarContainer = styled.div<{ justify : string}>`
  position: absolute;
  width: 100%;
  display: flex;
  padding: 0px 24px;
  z-index: 1;
  justify-content: ${({ justify }) => justify}
`;
```

And then, in the tsx part, you could call:

```tsx
<AvatarContainer justify={layoutName === 'small' ? 'center' : 'unset'}>
```

// Just saw that you know this is a thing. Sorry for the explanation 😅.

Reply:
I just saw this today! 😄 I appreciate the explanation. It's a much better approach.